suite-common/walletconnect/src/adapters/index.ts

```
@@ -18,26 +20,17 @@ export const getAdapterByNetwork = (networkType: string) =>
-        if (!eip155.chains.includes(walletConnectChainId)) {
-            eip155.chains.push(walletConnectChainId);
-        }
-        eip155.accounts.push(`${walletConnectChainId}:${account.descriptor}`);
-    });
-
-    return { eip155 };
-};
+export const getNamespaces = (accounts: Account[]) =>
+    adapters
+        .map(adapter => adapter.getNamespace(accounts))
+        .reduce((acc, val) => ({ ...acc, ...val }), {});
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)